bayem/distributions.py

```diff
@@ -0,0 +1,184 @@
+import numpy as np
+from scipy.optimize import brenth
+from scipy.stats import gamma, multivariate_normal, norm
+from tabulate import tabulate
+
+
+class MVN:
+    def __init__(self, mean=[0.0], precision=[[1.0]], name="MVN", parameter_names=None):
+        """
+        Creates a N-dimensional multivariate normal distribution from provided 
+        `mean` μ and `precision` L with the PDF
+
+           f(x) = (2π)^(-N/2) det(L)^(1/2) exp[-1/2 (x - μ)^T L (x - μ)]
+
+        mean:
+            vector of N means μ
+
+        precision:
+            precision matrix L with dimension NxN
+
+        name:
+            name of the distribution, e.g. to indicate "prior" or "posterior"
+
+        parameter_names:
+            vector of N strings that name the N parameters for convenient 
+            access via `self.named_mean`
+        """
+        self.mean = np.atleast_1d(mean).astype(float)
+        self.precision = np.atleast_2d(precision).astype(float)
+        self.name = name
+        if parameter_names is None:
+            self.parameter_names = [f"p{i}" for i in range(len(self.mean))]
```

Comment:
Minor:
maybe to use:
~~~py
self.parameter_names = [f"self.name-{i}" for i in range(len(self.mean))]
~~~

Reply:
Well, according to the paper notation, those would better be called `θ_{i}` or `$\theta_{i}$`. Could you live with that, what would you prefer?

Reply:
The `\theta` latex thing looks _much_ better in the provided `visualization` methods. So I used that for now. Are you fine with that?

Reply:
Yes sure. As I wrote in my comment. It was just a minor thing (suggestion).